packages/odyssey-react/src/components/Box/Box.test.tsx

```diff
@@ -27,6 +27,12 @@ describe("Box", () => {
     expect(screen.getByText("box").tagName.toLowerCase() === "span").toBe(true);
   });
 
+  it("allows a custom class", () => {
+    render(<Box className="testClass">box</Box>);
```

Comment:
hmmm I'm surprised this works because of how [our polymorphic types were defined here](https://github.com/okta/odyssey/blob/develop/packages/odyssey-react/src/utils/polymorphic.ts#L21). That utility method removes the `style` and `className` members from the shape of the props argument but perhaps there is something else happening here?